development/prow-installer/pkg/kubectl/wrapper.go

```diff
@@ -0,0 +1,104 @@
+package kubectl
+
+import (
+	"fmt"
+	log "github.com/sirupsen/logrus"
+	"os"
+	"os/exec"
+	"path/filepath"
+)
+
+type Wrapper struct {
+	Kubeconfig string
+}
+
+func (c *Wrapper) Apply() *KubeApplyCommand {
+	return &KubeApplyCommand{Config: c.Kubeconfig}
+}
+
+type KubeApplyCommand struct {
+	Type   string
+	Path   string
+	Nspace string
+	Config string
+}
+
+func (kac *KubeApplyCommand) File(path string) *KubeApplyCommand {
+	kac.Type = "-f" //file
+	//change slashed path (dir/filename) to OS-dependent path.
+	kac.Path = filepath.FromSlash(path)
```

Comment:
Not sure what the `filepath.FromSlash()` function is supposed to do, but I'm not sure what it does at all: https://play.golang.org/p/0ioTHQPHn2S (https://pkg.go.dev/path/filepath?tab=doc#FromSlash)
If path is a directory, you could just check if the directory exists:
```
if stat, err := os.Stat("/path/"); !os.IsNotExist(err) && stat.IsDir() {
	// path exists
}
```

Reply:
`filepath.FromSlash()` returns path with os-dependent separator. That means regardless the system the path should be valid for OS.

for `File()` setter the path can be either relative or absolute path to a file. I can add a checker if the given path exists and is not a directory.